DP02_17_Galaxy_Photometry.ipynb

```diff
@@ -0,0 +1,1077 @@
+{
```

Comment:
<p>Only g band and i band data are used in this plot. But u band and r band data are included in the query and magnitude calculation. Do you plan to show u band and r band in a different plot? Or leave them as exercises for learners? </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rubin-dp0/tutorial-notebooks/pull/231/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='50'/>

Reply:
<p>Good catch. this was leftover from earlier experimentation. I have now fixed it</p>